hs_communities/views/communities.py

```diff
@@ -38,8 +39,14 @@ def get_context_data(self, **kwargs):
 
         groups = sorted(groups, key=lambda key: key['name'])
 
-        u = User.objects.get(pk=self.request.user.id)
-        is_admin = u.username == 'czo_national'
```

Comment:
This is great... What about the checks on czo_national in the topicsview below in both post and get?  Is this something we can address now, or would you like to wait?

Reply:
Alas, topics view is custom for the CZOs and would take a lot to generalize. I can't fix it now. I'll put it on the list for when we make community functionality generally accessible. 

The code I changed is -- by contrast -- used by all communities.